pkg/virt-config/feature-gates.go

```diff
@@ -85,13 +85,8 @@ const (
 
 func (config *ClusterConfig) isFeatureGateEnabled(featureGate string) bool {
 	deprecatedFeature := deprecation.FeatureGateInfo(featureGate)
-	if deprecatedFeature != nil {
```

Comment:
It sounds odd that it is allowed to be added to the list, why not to reject usage of a FG in discontinue state?

Reply:
It may break existing workloads as the reconcile at several places re-checks the FG.
If someone removed the FG and then is unable to add it, it will be problematic.

As I see it, we should not limit the FG list, we should just make sure new VMs will not start.
If someone has a reasoning to make this stricter by not allowing to add the FG to the list, then I suggest it to come as a follow up after a thorough discussion on the subject.